server/src/main/java/pl/edu/agh/RaftServer.java

```diff
@@ -108,8 +129,7 @@ public State getState() {
                     return Optional.of(response);
                 }),
                 Case(instanceOf(AppendEntries.class), ae -> {
-                    // TODO: check content and perform proper actions
```

Comment:
That's what I thought. I wanted to make it in compliance with the paper (page 4) - empty AppendEntries for heartbeats, filled for replication. Could we change that?